docs/install/Knative-custom-install.md

```diff
@@ -304,23 +304,29 @@ commands below.
    completed until the upgrade process finishes.
 
 1. To install Knative components or plugins, specify the filenames in the
-   `kubectl apply` command:
+   `kubectl apply` command.
 
-   - To install an individual component or plgugin
+   - Run the `kubectl apply` command once with the
+   `-l knative.dev/crd-install=true` flag to install the CRDs first. (This
+   prevents race conditions, which cause intermittent install errors.) 
 
      ```bash
-     kubectl apply --filename [FILE_URL]
+     kubectl apply -l knative.dev/crd-install=true \
+     --filename [FILE_URL] \
+     --filename [FILE_URL]
      ```
 
-   - To install multiple components or plugins, append additional
-     `--filename [FILENAME]` flags to the `kubectl apply` command:
+   - Then run the `kubectl apply` command without the `-l` flag to complete the
+   install:
 
      ```bash
-     kubectl apply --filename [FILE_URL] --filename [FILE_URL] \
-       --filename [FILE_URL]
+     kubectl apply --filename [FILE_URL] \
+     --filename [FILE_URL]
      ```
 
-     where [`FILE_URL`] is the URL path of the desired Knative release:
+     You can add as many `--filename [FILE_URL]` as needed.
+
+     [`FILE_URL`] is the URL path of the desired Knative release:
```

Comment:
I think that wording was from your original version of this guide, and it seems clear (and concise) to me.

Reply:
today we have multiple components and plugins available so its no longer accurate but since you plan to recover other items in a separate PR, we can go that route